mesa/visualization/components/matplotlib.py

```diff
@@ -63,7 +63,9 @@ def SpaceMatplotlib(
     elif space is None and propertylayer_portrayal:
         draw_property_layers(space_ax, space, propertylayer_portrayal, model)
 
-    solara.FigureMatplotlib(space_fig, format="png", dependencies=dependencies)
+    solara.FigureMatplotlib(
+        space_fig, format="svg", bbox_inches="tight", dependencies=dependencies
```

Comment:
Using png is intentional: https://github.com/projectmesa/mesa/issues/1806#issue-1899325310. At the very least, a comment should be added to say it is for performance reason.

Reply:
Yes svg is indeed very low as compare to png: [solara-perf.ipynb](https://github.com/wang-boyu/mesa-examples/blob/solara-perf/examples/schelling_experimental/solara-perf.ipynb) (863 ms for png vs. 3.89 s  for svg, for a 200x200 schelling model). I also tried increasing png dpi from default 100 to 300, and it becomes slower with no noticeable improvement in terms of plot quality. I have changed back to png format in this PR.